docs/containers-registries.conf.5.md

```diff
@@ -77,6 +82,18 @@ internet without having to change `Dockerfile`s, or to add redundancy).
     requests for the image `example.com/foo/myimage:latest` will actually work with the
     `internal-registry-for-example.net/bar/myimage:latest` image.
 
+    With a `prefix` containing a wildcard in the format: "*.example.com" for subdomain matching,
+    the location can be empty. In such a case,
+    prefix matching will occur, but no reference rewrite will occur. The
+    original ref will be used as-is.
+
+    Example: Given
+    ```
+    prefix = "*.example.com"
+    ```
+    requests for the image `blah.example.com/foo/myimage:latest` will be used
+    as-is. 
```

Comment:
(Maybe something like “but other settings like `insecure`/`blocked`/`mirrors` will be applied to matching images” to explain why one would do this.)

Reply:
ack, will add ...

Reply:
addressed in the latest version of the commit. Marking as resolved.